aten/src/THCUNN/ClassNLLCriterion.cu

```diff
@@ -125,7 +125,7 @@ __global__ void cunn_ClassNLLCriterion_updateOutput_kernel(Dtype *output,
     }
     *total_weight = ScalarConvert<Acctype, Dtype>::to(total_weightAcc);
     *output = ScalarConvert<Acctype, Dtype>::to(outputAcc);
-    if (size_average && *total_weight > 0) {
+    if (size_average && *total_weight >= 0) {
```

Comment:
There's some code in  cunn_ClassNLLCriterion_updateOutput_kernel1 that does the same thing, do we need to update that? https://github.com/pytorch/pytorch/blob/638e4ad8b95793876c0f5e82db58c18618fb6e61/aten/src/THCUNN/ClassNLLCriterion.cu#L35-L37